datamodel/src/main/java/com/hartwig/serve/datamodel/EvidenceDirection.java

```diff
@@ -1,24 +1,31 @@
 package com.hartwig.serve.datamodel;
 
 public enum EvidenceDirection {
-    RESPONSIVE(true, false, true),
-    PREDICTED_RESPONSIVE(true, false, false),
-    NO_BENEFIT(false, false, true),
-    RESISTANT(false, true, true),
-    PREDICTED_RESISTANT(false, true, false);
+    RESPONSIVE(true, false, false, true),
+    PREDICTED_RESPONSIVE(true, false, false, false),
+    DECREASED_RESPONSE(false, false, false, true),
+    NO_BENEFIT(false, true, false, true),
+    RESISTANT(false, true, true, true),
+    PREDICTED_RESISTANT(false, true, true, false);
 
-    private final boolean isResponsive;
+    private final boolean hasPositiveResponse;
+    private final boolean hasNoBenefit;
     private final boolean isResistant;
     private final boolean isCertain;
 
-    EvidenceDirection(final boolean isResponsive, final boolean isResistant, final boolean isCertain) {
-        this.isResponsive = isResponsive;
+    EvidenceDirection(final boolean hasPositiveResponse, final boolean hasNoBenefit, final boolean isResistant, final boolean isCertain) {
+        this.hasPositiveResponse = hasPositiveResponse;
+        this.hasNoBenefit = hasNoBenefit;
```

Comment:
It is more typical to define booleans in terms of their "positive" intent, so this variable should then be called "hasBenefit". Or maybe a better name would be "impliesResponsivenessOrResistance`? or `impliesImpact`?

Reply:
I personally prefer `hasBenefit`. Some background on my logic for the booleans:
- `hasPositiveResponse` makes sure we can distinct "sensitive" (positive response) from "decreased response", "resistant", "no benefit" (negative response).
- Then `isResistant `makes a distinction between "resistant" (resistant) and "no benefit" & "decreased response" (no resistance).
- Lastly `hasBenefit` makes a distinction between "no benefit" (has no benefit) and "decreased response" (has benefit)